docs/source/resources/frequently_asked_questions.ipynb

```diff
@@ -1196,7 +1196,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Finally, you might only want features that return a certain type. You can do this by using the `column_schema` attribute."
+    "Finally, you might only want features that return a certain type. You can do this by using the `column_schema` attribute. For more information on working with column schemas, take a look at [Transitioning from Variables to Woodwork](docs/source/resources/transition_to_ft_v1.0.ipynb)."
```

Comment:
I'm never entirely comfortable with how links work in the docs builds, but the new transition guide will be in the resources folder as well, so I think you can maybe just link to `transition_to_ft_v1.0.ipynb` here instead of `docs/source/resources/transition_to_ft_v1.0.ipynb`.